spec/controllers/application_controller_spec.rb

```diff
@@ -534,7 +534,7 @@ def append_info_to_payload(payload)
     end
 
     context 'when the current user and session object exist' do
-      it 'returns a hash with the user and session info' do
+      it 'returns the current user session token' do
         get :test_authentication
```

Comment:
I just noticed this from the previous PR, can you remove the line `get :test_authentication` ? The actual route call is getting called in `subject`, so this could represent a (minor) test inaccuracy

Reply:
I removed this and I got an error with `ttl_in_time` not being defined. This makes sense since it looks like the object this was on was set  with the erroneous `test_authentication` call. What you can do to get the `expiration_time` itself is something like this:

In a `before do` block, add `Timecop.freeze`
In a `after do` block, add `Timecop.return`
Set `expiration_time` to `(Time.zone.now + 30.minutes).iso8601` (or something like that, this should be close at least)

Reply:
I removed get :test_authentication. Then, used Session.find(token).ttl_in_time.iso8601(0). No issues with ttl_in_time. So, there wasn't a need to use Timecop.freeze and Timecop.return.

Reply:
Sounds good, looks like that worked!